wagtail/admin/templates/wagtailadmin/bulk_actions/confirmation/form_with_fields.html

```diff
@@ -0,0 +1,9 @@
+{% extends './form.html' %}
+
+{% block form_fields %}
+<ul class="fields">
```

Comment:
Looks like this is pretty similar to the `form_as_ul` template - I would suggest reusing the existing one